### ISSUE TITLE
Error Uploading to App Store

### ISSUE THREAD

**Issue by julioarregoitia** [getbouncer/cardscan-ios#260]
I adapted CardScan from Carthage to Swift Package Manager, removed the reference to `/ usr / local / bin / carthage copy-frameworks` that belonged to` CardScan` now when uploading the app the next error appears. (see the next image)

If you can, try including CardScan with Swift Package Manager in an app and try uploading the Build to the App Store
<img width="629" alt="Captura de Pantalla 2021-02-09 a la(s) 6 16 23 p  m" src="https://user-images.githubusercontent.com/35386842/107441250-ed48a600-6b02-11eb-88b9-6a8e7c419737.png">

**Comment by jaimejiyepark** [comment]
Hello @julioarregoitia, 
Let take a look for you. Can you tell me what version of `CardScan` you are trying to use?
Also from the looks of the first build error, it looks `CardScan.framework` is not in the folder `Frameworks`. If you haven't already, please give [this solution]( https://www.simpleswiftguide.com/how-to-add-xcframework-to-xcode-project/) a try!

**Comment by julioarregoitia** [comment]
First of all thanks for answering:
- We use version 2.0.7
- I will try the .xcframework solution (although I use other Frameworks with SPM and this error had not happened to me, but I will try and tell them)
- Yes, the entire Derived Data folder was deleted, the Mac was restarted and everything was compiled again

<img width="287" alt="Captura de Pantalla 2021-02-09 a la(s) 6 54 17 p  m" src="https://user-images.githubusercontent.com/35386842/107444251-39e2b000-6b08-11eb-9629-4df9dca11745.png">

**Comment by julioarregoitia** [comment]
I reread the CardScan installation instructions (https://docs.getbouncer.com/bouncer-scan/credit-card-ocr/ios-integration-guide#installation) for `SPM` and made sure I was doing this, at the end of the process `CardScan` is added as shown in the image. What is the relationship with .xcframework for your case ???

<img width="700" alt="Captura de Pantalla 2021-02-09 a la(s) 7 04 27 p  m" src="https://user-images.githubusercontent.com/35386842/107444990-a4482000-6b09-11eb-82b6-2ae685f69c23.png">

**Comment by julioarregoitia** [comment]
By the way, in this part of here you quote `https://github.com/getbouncer/cardscan-ios/git` , but I got an error with this. I think must be `https://github.com/getbouncer/cardscan-ios.git` , it is a *dot* before the *git* word

<img width="749" alt="Captura de Pantalla 2021-02-09 a la(s) 7 18 03 p  m" src="https://user-images.githubusercontent.com/35386842/107445989-8bd90500-6b0b-11eb-8eea-95049764bb46.png">

**Comment by jaimejiyepark** [comment]
Thank you for pointing error out in the docs, I will fix it ASAP. Can you check `Build Phase -> Link Binary with Libraries` to see if `CardScan` is there? Also I will provide an example app below. Please let me know if you can run it on your machine. 

[CarthageBuildTest.zip](https://github.com/getbouncer/cardscan-ios/files/5955010/CarthageBuildTest.zip)

**Comment by julioarregoitia** [comment]
The example run it here

![Captura de Pantalla 2021-02-09 a la(s) 7 40 08 p  m](https://user-images.githubusercontent.com/35386842/107447929-9f399f80-6b0e-11eb-981b-db9994bb4325.png)

**Comment by julioarregoitia** [comment]
And in my project `CardScan` also exist in the path `Build Phase -> Link Binary with Libraries`
Any other idea??

![Captura de Pantalla 2021-02-09 a la(s) 7 41 09 p  m](https://user-images.githubusercontent.com/35386842/107447993-c3957c00-6b0e-11eb-806d-d7c39a82670d.png)

**Comment by kingst** [comment]
Hi @julioarregoitia let me lean in to help out with this. To answer your earlier question, yes, we've had many apps that have shipped the latest version of our SDK to the app store, so we do know that it's possible and I just uploaded one of our test apps that uses SPM to double check.

Based on the error message that you posted originally it looks like there are two copies of CardScan in your app right now, if I'm reading that correctly. Can you look through the directory where xcode stores your archives to see if there are two copies of CardScan.framework there?

**Comment by julioarregoitia** [comment]
I already deleted the entire `DerivedData` folder that contained all the files that Xcode stores. I checked in my project folder if there was still any `CardScan` and nothing there either.
I reopened the project to load all the SPM files and still when doing the ** Archive ** it appears double `CardScan` as you can see in the next image  🤷‍♂️🤷‍♂️

<img width="746" alt="Captura de Pantalla 2021-02-10 a la(s) 3 56 49 p  m" src="https://user-images.githubusercontent.com/35386842/107572463-54756180-6bba-11eb-8253-db7fff2460cb.png">

**Comment by julioarregoitia** [comment]
I really don't understand what is the problem your project has with `SPM`. I removed the version of `SPM` and added the manual project and so if it works well as you can see in the images.
**I repeat, it has not been fixed with `SPM`, if manual is added it works fine, if it is added with` SPM` it does not work**

<img width="744" alt="Captura de Pantalla 2021-02-10 a la(s) 4 44 53 p  m" src="https://user-images.githubusercontent.com/35386842/107576560-4ece4a80-6bbf-11eb-86cc-a468cec207ca.png">
<img width="251" alt="Captura de Pantalla 2021-02-10 a la(s) 4 46 18 p  m" src="https://user-images.githubusercontent.com/35386842/107576717-8210d980-6bbf-11eb-86b9-b3967f384b07.png">
<img width="744" alt="Captura de Pantalla 2021-02-10 a la(s) 4 47 34 p  m" src="https://user-images.githubusercontent.com/35386842/107576832-aff61e00-6bbf-11eb-88fb-fea1ed4f1b09.png">

**Comment by kingst** [comment]
Can you open the `Carthage` directory to see if there is still a stray copy of CardScan there?

**Comment by julioarregoitia** [comment]
Thanks guys, I don't know what it is. I made the `Archive` of your example and it had no problem.
  In the project folder there was nothing of `CardScan`, in` DerivedData` neither is there.
I went to ** plan B ** and downloaded the source code, compiled it with ** Carthage ** (which now allows generating the .XCFrameworks) and added it to the project. Thanks for your support and effort, I don't know what the problem will be with `SPM` but we are not going to waste any more time with something that I don't understand what it is.
